pkg/proxy/request.go

```diff
@@ -30,9 +33,11 @@ func httpValidator(r *http.Request) url.Values {
 		"type":            []string{"required"},
 		"overwrite":       []string{"in:1,True,true,t,T,TRUE,0,False,false,f,F,FALSE"},
 		"distributedPods": []string{"numeric_between:1,"},
-		"file:testFile":   []string{"ext:jmx"},
+		"file:testFile":   []string{"ext:jmx,py"},
```

Comment:
This way we can create Locust test with .jmx file instead of .py because both file types are valid. But it doesn't make sense. And the test will fail only after creating all the resources and failing on start. 
Let's maybe add validation of file type to fromHTTPRequestToLoadTestSpec?

Reply:
It would not fail, as after reading the file it will be handled as string the whole time.
I would prefer to remove file extension validation at all, as we are not actually protecting anything.
And then, on a different PR, implement mimetype validation and some sort of per backend rules.

Reply:
Agree with @diegomarangoni  about removing file validation, doesn't make sense. All validation logic should happen on the spec side of the backend.